react/features/authentication/middleware.ts

```diff
@@ -268,16 +270,16 @@ function _handleLogin({ dispatch, getState }: IStore) {
         return;
     }
 
-    // FIXME: This method will not preserve the other URL params that were originally passed.
-    const tokenAuthServiceUrl = getTokenAuthUrl(config, room);
+    getTokenAuthUrl(config, room, tenant, true)
+        .then((tokenAuthServiceUrl: string | undefined) => {
```

Comment:
Can you please test everything works ok since this now returns a promise? Looks ok but just to be sure!